roles/nextflow/defaults/main.yml

```diff
@@ -1,3 +1,3 @@
 nextflow_java: "/sw/comp/java/x86_64/sun_jdk1.8.0_151"
-nextflow_version_tag: "v20.01.0"
+nextflow_version_tag: "v20.10.0"
```

Comment:
I think that rnaseq v3 needs the latest edge release. However, that was to fix a problem with singularity image download I think so hopefully should be fine with this version. Bear in mind for testing though 👍🏻

Reply:
We'll have a validation with the stage deployment first, so if `v20.10.0` doesn't work out, we can update to the latest edge release and test again.

Reply:
Sounds good. I think that the next stable release should be out pretty soon - in a week or two maybe.